manifests/golang.yml

```
@@ -461,6 +461,7 @@ tests/:
       Test_SecurityEvents_Iast_Metastruct_Disabled: irrelevant (No IAST)
       Test_SecurityEvents_Iast_Metastruct_Enabled: irrelevant (No IAST)
     test_remote_config_rule_changes.py:
+      Test_AsmDdMultiConfiguration: missing_feature
```

I think this is implemented in `v2.1.0-dev` (if not in `v2.0.0`, but I reckon it's not there).

```suggestion
      Test_AsmDdMultiConfiguration: v2.1.0-dev
```